### ISSUE TITLE
OnMouseOver does not work on Chrome

### ISSUE THREAD

**Issue by rgaiacs** [facebook/react#5577]
I'm using react 0.14.3 and have something like

~~~
render() {
    return <canvas onMouseMove={this.props.trackMousePosition.bind(this)} />
}

trackMousePosition(event:Object) {
    console.log("Change: " + event.clientX + ", " + event.clientY);
}
~~~

On Firefox (44.0a1) the `onMouseMove` works but on Chrome 46.0.2490.86 (64-bit) doesn't.

**Comment by rgaiacs** [comment]
## Minimal working example

~~~
import React from 'react/addons';

var attachElement = document.getElementById('main');

var divStyle = {
    backgroundColor: 'red',
    width: 100 + 'px',
    height: 100 + 'px',
};


React.render(
        <div
            onClick={() => {console.log("onClick");}}
            onDoubleClick={() => {console.log("onDoubleClick");}}
            onMouseEnter={() => {console.log("onMouseEnter");}}
            onMouseLeave={() => {console.log("onMouseLeave");}}
            onMouseMove={() => {console.log("onMouseMove");}}
            style={divStyle} >
        </div>,
        attachElement);
~~~

## Gecko

E.g. Firefox.

Event | Status
------ |------
onClick | OK
onDoubleClick | OK
onMouseEnter | OK
onMouseLeave | OK
onMouseMove | OK

## WebKit

E.g. Epiphany.

Event | Status
------ |------
onClick | OK
onDoubleClick | OK
onMouseEnter | OK
onMouseLeave | OK
onMouseMove | OK

## Blink

E.g. Chrome/Chromium.

Event | Status
------ |------
onClick | OK
onDoubleClick | OK
onMouseEnter | **Fail.** Only work when mouse's left button is press.
onMouseLeave | **Fail.** Only work when mouse's left button is press.
onMouseMove | **Fail.** Only work when mouse's left button is press.

**Comment by rgaiacs** [comment]
I'm closing this since it should since this is need to be fixed upstream: https://code.google.com/p/chromium/issues/detail?can=2&start=0&num=100&q=onmousemove&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Cr%20Status%20Owner%20Summary%20OS%20Modified&groupby=&sort=&id=564085

**Comment by jimfb** [comment]
My fiddle: http://jsfiddle.net/ucrjxjv5/

FWIW, I'm unable to reproduce this issue on my chrome/linux (Chrome Version 47.0.2526.73 (64-bit)).  Linux ubuntu 3.16.0-50-generic #67~14.04.1-Ubuntu SMP Fri Oct 2 22:07:51 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux.

**Comment by brandonrninefive** [comment]
I just encountered this bug myself on Chrome Version 63.0.3239.108 on Debian 9, kernel version 4.9.0-amd64. The events are only firing when the mouse is clicked, and not on enter/exit of an element with a mouse enter/leave listener. This happens to me on the jsfiddle in the previous comment as well. I noticed that the Chromium issue posted above has been closed too, but I can confirm this issue still exists. Switching to Firefox produced the correct behavior.
